pornography-hosts

```diff
@@ -15278,3 +15278,6 @@
 0.0.0.0 gotxx.com
 0.0.0.0 mydesi.net
 0.0.0.0 nuefliks.club
+0.0.0.0 desixflix.com
```

Comment:
https://github.com/Clefspeare13/pornhosts/blob/a407e2cd7f662ea9481242e00d14db42df771233/submit_here/hosts.txt#L2934

Reply:
I'm using StevenBlack pornhosts list, and the site isn't blocked there.

Reply:
Yeah, that's because of https://github.com/funilrys/PyFunceble/issues/243 

Since I haven't been able to recreate the lists, you don't see it inside any of the lists in https://github.com/Clefspeare13/pornhosts/tree/master/download_here

Reply:
Hi, I'll remove this then from here. Do you co own this repo btw? Coz I initially thought it's independent of the changes in your repo. The reason I commit changes here is because it's difficult for me to add sites in bulk to your repo. Since I generally find more than one site, and it's difficult for me to fill a form for each one.

Reply:
Hey @defa02005

This project is solely owned by @Sinfonietta my comments have a couple of reasons

1. Sharing information according to my projects main goal, ( One index (issue pr domain) for shared knowledge)
2. No reason to add domains several time in several projects, just to have them clean at the end
3. Sometimes somebody find something other have overlooked . Or just things have changed.

> The reason I commit changes here is because it's difficult for me to add sites in bulk to your repo.

Yes I don't support that for the reasons just mentioned. Therefor one issue = one domain, to keep as simple a index and track of changes, this is the best long term way to do it.

> Since I generally find more than one site, and it's difficult for me to fill a form for each one.

They should be made rather simple to support copy/paste...  And more data is = faster committed to the lists.

Please commit suggestions to the [templates](https://mypdns.org/my-privacy-dns/porn-records/-/tree/master/.gitlab/issue_templates) nothing is ever `the best`, there are always room for good idea and improvements.

> I'll remove this then from here

All this said, I believe you should let @Sinfonietta decide by himself :wink: